src/PatientChecking/PatientChecking/Services/ServiceModels/Enum.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientChecking.Services.ServiceModels
+{
+    public class Enum
```

Comment:
We don't need `Enum` class, just:
```
public enum AppointmentStatus { 
CheckIn = 1, 
Closed = 2, 
Cancel = 3 
}
```
Please rename `StatusAppointment` to `AppointmentStatus`

Reply:
I separated Appointment Status into enum folder